pkg/remediator/queue/queue.go

```diff
@@ -73,12 +73,12 @@ type ObjectQueue struct {
 	// also allow any call to yield the lock safely (specifically for Get).
 	cond *sync.Cond
 	// rateLimiter enables the ObjectQueue to support rate-limited retries.
-	rateLimiter workqueue.RateLimiter
+	rateLimiter workqueue.TypedRateLimiter[GVKNN]
```

Comment:
Are these generic types available in the previous go version? 

If so, can you move this out to a different PR, just to reduce the size of this one.

If not, no worries.

Reply:
Looks like these types aren't available in the previous k8s version, so it needs to be changed at the same time